code/drasil-lang/Language/Drasil/DisplayExpr.hs

```diff
@@ -7,7 +7,14 @@ import Language.Drasil.Expr (Expr)
 import Language.Drasil.Space (Space)
 
 data DisplayBinOp = Defines | IsIn
-data DisplayAssocBinOp = And
+  deriving (Eq, Show)
+
+-- | Associative Binary Display Operations
+--
+--   Note: 'Equal' makes sense in the Display language, but not in the
+--         general computation language.
+data DisplayAssocBinOp = And | Equal
+  deriving (Eq, Show)
```

Comment:
Well, on second thought, 'Equal' may not be good for the associative list.

Reply:
In a sense, it can be, but in another, it's definitely not. I'm starting to confuse myself with it. Any thoughts on this @JacquesCarette?

Reply:
It's indeed quite tricky. Let's call it `Equivalence` instead, that will be less confusing.

Reply:
Thanks! I changed the name now.